overlord/devicestate/handlers_install.go

```diff
@@ -580,6 +583,19 @@ func (m *DeviceManager) doRestartSystemToRunMode(t *state.Task, _ *tomb.Tomb) er
 		return fmt.Errorf("missing modeenv, cannot proceed")
 	}
 
+	preseeded, err := maybeApplyPreseededData(boot.InitramfsUbuntuSeedDir, modeEnv.RecoverySystem, boot.InstallHostWritableDir)
+	if err != nil {
+		msg := fmt.Sprintf("failed to apply preseed data: %v", err)
+		t.Logf(msg)
+		logger.Noticef(msg)
```

Comment:
Do we need both of these? Just asking, because I see that in other error scenarios we just call `logger.Noticef()`. If `t.Logf()` is also needed, then the question becomes whether it should be added to the other cases below. :-)